src/control/drpc/drpc.pb.go

```diff
@@ -5,8 +5,9 @@ package drpc
 
 import (
 	fmt "fmt"
-	proto "github.com/golang/protobuf/proto"
 	math "math"
+
+	proto "github.com/golang/protobuf/proto"
```

Comment:
I would remove this change. There is no reason for us to be modifying the pre-generated code.